### PR TITLE
RF | RenamePipelineTest > all tests

### PR DESCRIPTION
**Pull request by maratdp**
refactor and move all tests to _PipelineTest class;
delete RenamePipelineTest class

### CODE REVIEW

**Comment by maratdp** [comment]
[RF | RenamePipelineTest > all tests](https://trello.com/c/JZpKhSGk/856-rf-renamepipelinetest-all-tests)

**Review by Stanislaw1349 — APPROVED** [review]
LGFM

File: src/test/java/_PipelineTest.java [hunk 1]
```diff
@@ -713,4 +729,74 @@ public void testRenamePipelineTheSameNameWithAllCapitalLetters() {
         Assert.assertEquals(actualWarning,"The name “GENERAL” is already in use.");
         Assert.assertEquals(actualError, "Error");
     }
+
+    @Test
+    public void testRenamePipelineWithValidName() {
+        final String pipelineName = pipelineName();
+        final String newPipelineName = "NEW" + pipelineName;
+
+        homePageClick();
```
Comment (OVFgit) [inline src/test/java/_PipelineTest.java:738]: There is special method for this click in ProjectUtils class

Comment (maratdp) [reply src/test/java/_PipelineTest.java:738]: Thank you. Will update with new commit

**Pull request merged**